runtime/moonbase/src/governance/tracks.rs

```diff
@@ -154,6 +154,20 @@ const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 8]
 			min_support: Curve::make_reciprocal(8, 14, percent(1), percent(0), percent(10)),
 		},
 	),
+	(
+		16,
+		pallet_referenda::TrackInfo {
+			name: "general_admin",
+			max_deciding: 5,
+			decision_deposit: 30 * KILOUNIT * SUPPLY_FACTOR,
+			prepare_period: 4,
+			decision_period: 14 * DAYS,
+			confirm_period: 48 * HOURS,
+			min_enactment_period: 1 * DAYS,
+			min_approval: Curve::make_linear(14, 14, percent(50), percent(100)),
+			min_support: Curve::make_reciprocal(8, 14, percent(1), percent(0), percent(10)),
```

Comment:
Ensure this matches [official track config](https://purestake.atlassian.net/wiki/spaces/~61d322af326554006ced8d63/pages/2430238725/Governance+track+proposal) like [Gov2 Precompiles PR](https://github.com/PureStake/moonbeam/pull/1885/files#diff-c76c6394502dad4b9ec16834960252bc6b99a57333fb0c0b242c7233e33078a0R157-R170)

Reply:
yeah these are temporary values, we just need to decide on these before we merge this pr